src/plugins/inspector/public/views/requests/_requests.scss

```diff
@@ -1,5 +1,5 @@
 .insRequestDetailsStats__icon {
-  margin-right: $euiSizeS;
+  margin-left: $euiSizeS;
```

Comment:
I think it looks a bit awkward with the larger gap now that the icons were moved to the right of the text - I wonder if an `$euiSizeXS` gap is better? cc @elastic/kibana-design 

| `$euiSizeS` | `$euiSizeXS` |
|--------|--------|
| ![image](https://github.com/elastic/kibana/assets/8698078/dbaed4c4-7ed2-4679-9849-9fe069306402) | ![image](https://github.com/elastic/kibana/assets/8698078/3d373215-351e-4981-b31c-272dfcc982e3) |